src/lib/adapter.ts

```diff
@@ -54,6 +55,7 @@ class GleeAdapter extends EventEmitter {
     this._parsedAsyncAPI = parsedAsyncAPI
     this._channelNames = this._parsedAsyncAPI.channels().all().map(e => e.id())
     this._channelAddresses = this._parsedAsyncAPI.channels().all().map(c => c.address())
+    this._operationIds = this._parsedAsyncAPI.operations().all().map(o => o.operationId())
```

Comment:
In v3, there's no such thing as `operationId` so I'm betting that it will get removed pretty soon from the parser API. I'd instead rely on `.id()`:

```suggestion
    this._operationIds = this._parsedAsyncAPI.operations().all().map(o => o.id())
```